database/migrations/2025_06_07_000000_update_event_achievements_table.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    public function up(): void
+    {
+        Schema::table('event_achievements', function (Blueprint $table) {
+            $table->string('decorator', 96)->nullable()->after('active_until');
```

Comment:
96 is even longer than what we allow for achievement titles. Recommend capping this at something lower - maybe 64?

Reply:
I assumed Title was longer than that. I've changed this to 40.